packages/rulesets/generated/spectral/az-arm.js

```diff
@@ -2690,6 +2690,27 @@ const validatePatchBodyParamProperties = createRulesetFunction({
     return errors;
 });
 
+const requestBodyMustExistForPutPatch = (putPatchOperationParameters, _opts, ctx) => {
+    const errors = [];
+    const path = ctx.path;
+    const error = `The put or patch operation does not have a request body defined. This is not allowed. Please specify a request body for this operation.`;
+    const bodyParam = findBodyParam(putPatchOperationParameters);
+    if (bodyParam == undefined || bodyParam["schema"] == undefined) {
+        errors.push({
+            message: error,
+            path: path,
+        });
+    }
+    return errors;
+};
+function findBodyParam(params) {
```

Comment:
You can probably put this in utils.ts

Reply:
It is in utils. This is the generated code. @bdefoy